docs/src/views/AccessibilityBehaviors.tsx

```diff
@@ -0,0 +1,256 @@
+      <li>{link('ARIA attributes', '/accessibility-behaviors#aria-attributes')}</li>
+      <li>{link('Overriding behaviors', '/accessibility-behaviors#overriding-behaviors')}</li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      In Stardust, accessibility behaviors encapsulate keyboard and screen reader navigation. They
+      essentially add ARIA roles, ARIA attributes and event handlers to components' parts. The idea
+      is to compose visual components and apply a behavior on top of them to achieve desired
+      keyboard and screen reader navigation.
+    </p>
+    <p>
+      Each relevant component comes with its default accessibility behavior. For some components
+      there are additional behaviors to choose from. In addition to that, user can create custom
+      behaviors and use them instead of the standard ones.
+    </p>
+    <p>
+      Users are also able to add or override attributes generated by the accessibility behaviors if
+      customization is needed.
+    </p>
+    <p>
+      Accessibility behavior type is a callback function which receives "props" as parameters and
+      return an accessibility behavior object. Type:{' '}
+      {code('(props: any) => AccessibilityDefinition')}.
+    </p>
+
+    <div>
+      This {code('AccessibilityDefinition')} object consist of next properties which can be set in
+      behavior:
+      <ul>
+        <li>
+          <b>attributes</b> - ARIA or any other attributes that can be applied to component's parts.
+          <p>Type: {code('{ [partName: string]: AccessibilityAttributes }')}.</p>
+        </li>
+        <li>
+          <b>keyActions</b> - Actions that can be executed when specified keys are pressed. Action
+          names are mapped to action handlers in component and can be also assigned to particular
+          component's parts.
+          <p>Type: {code('{ [partName: string]: { [actionName: string]: KeyAction } }')}.</p>
+        </li>
+        <li>
+          <b>focusZone</b> - Focus Zone provides arrow key navigation within component's items, for
+          example, in menu, list, toolbar, grid etc. and at the same time it is possible to TAB
+          between these navigable components (navigate between Menu and List components by pressing
+          TAB and use arrow keys to navigate between their items).{' '}
+          {link('Read more about FocusZone.', '/focus-zone')}
+          <p>Type: {code('{ mode: FocusZoneMode, props?: FocusZoneProps }')}.</p>
+        </li>
+        <li>
+          <b>focusTrap</b> - FocusTrapZone grabs and traps focus within HTML content. If true, is it
+          enabled with default properties or can be modified by setting object.{' '}
+          {link('Read more about FocusTrapZone.', '/focus-trap-zone')}
+          <p>Type: {code('FocusTrapZoneProps | boolean')}.</p>
+        </li>
+        <li>
+          <b>autoFocus</b> - AutoFocusZone is used to grab focus and put it to inner element when
```

Comment:
maybe: AutoFocuxZone is used to isolate focus inside an inner element ...

Reply:
it eventually grabs focus, doesn't isolate it